cmd/controller/main.go

```diff
@@ -37,7 +41,7 @@ var (
 
 	address = flag.String("address", "0.0.0.0:8080", "The IP address and port to use for listening for client connections")
 
-	psqlConnection         = flag.String("psql-connection", "", "See https://pkg.go.dev/github.com/lib/pq#hdr-Connection_String_Parameters")
+	psqlConnection         = flag.String("psql-connection", "host=localhost port=5432 user=postgres password=password dbname=postgres sslmode=disable", "See https://pkg.go.dev/github.com/lib/pq#hdr-Connection_String_Parameters")
```

Comment:
This will break the check if we are requesting psql. In the OpenStorage function, I check if this is empty to determine whether we should use psql or memdb.

Reply:
done